packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -447,15 +447,29 @@ export class WorkspaceRootUriAwareCommandHandler extends UriAwareCommandHandler<
         super(selectionService, handler);
     }
 
+    isVisible(): boolean {
+        return !!this.workspaceService.tryGetRoots().length;
```

Comment:
Should not we call super.isVisible as well to make sure that handler validates a given uri? the same for `isEnabled`?